offset_fetch_request.go

```diff
@@ -2,10 +2,15 @@ package sarama
 
 type OffsetFetchRequest struct {
 	ConsumerGroup string
+	Version       int16
 	partitions    map[string][]int32
 }
 
 func (r *OffsetFetchRequest) encode(pe packetEncoder) (err error) {
+	if r.Version < 0 || r.Version > 1 {
```

Comment:
I'm not sure what you mean? Based on my reading of the protocol guide (and [Jun's comment in JIRA](https://issues.apache.org/jira/browse/KAFKA-1841?focusedCommentId=14270222&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14270222)) the version can be 0 or 1, so if it is `<0` or `>1` we need to bail.